descartes_planner/src/sparse_planner.cpp

```diff
@@ -56,7 +56,7 @@ descartes_core::TimingConstraint cumulativeTimingBetween(
 
 namespace descartes_planner
 {
-const int INVALID_INDEX = -1;
+const std::size_t INVALID_INDEX = -1;
```

Comment:
This may still work as intended, but we should probably change it to be `numeric_limits<std::size_t>::max()`. This is if this works.

@jrgnicho Can you comment on your code?

Reply:
**std::size_t** is an alias for an **unsigned integer** which can never be -1 so I'd suggest keeping it as an int

Reply:
I think we also need to modify `findNearestSparsePointIndex` and `getDensePointIndex `to return a `std::size_t` and this will be good. I'll update the pull request